contrib/git-sync/demo/config/service.yaml

```diff
@@ -1,8 +1,11 @@
-id: blog-service
+apiVersion: v1beta3
 kind: Service
-apiVersion: v1beta1
-port: 80
-containerPort: 80
-selector:
-  name: blog
-createExternalLoadBalancer: true
+metadata:
+  name: blog-service
+spec:
+  createExternalLoadBalancer: true
+  ports:
+  - port: 80
+    protocol: TCP
```

Comment:
You are right. 
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/v1beta3/types.go#L1082